cabal.project

```diff
@@ -330,6 +330,18 @@ source-repository-package
   tag: ee59880f47ab835dbd73bea0847dab7869fc20d8
   --sha256: 1lrzknw765pz2j97nvv9ip3l1mcpf2zr4n56hwlz0rk7wq7ls4cm
 
+source-repository-package
```

Comment:
Can we create an issue for the removal of this?

Reply:
I put the functions into `Cardano.CLI.IO.Lazy` for the moment.